.golangci.yml

```diff
@@ -6,3 +6,8 @@ linters:
 issues:
   max-issues-per-linter: 0
   max-same-issues: 0
+  exclude-rules:
+  - path: _test\.go
+    linters:
+    - staticcheck
+    text: "SA1019: tableprinter.NoHeaders is deprecated: use WithHeaders unless required otherwise."
```

Comment:
Small nit there is no newline at end of file, lets add that back in.

Reply:
This is odd. Both vim and vscode should be adding them anyway. But would it make sense to add a `.editorconfig` to the repo to help ensure it? A lot of editors (with or without extensions) support it.